pkgs/development/python-modules/pytest-leaks/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytest,
+  pytestCheckHook,
+  setuptools,
+  setuptools-scm,
+  wheel,
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-leaks";
+  version = "0.3.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "abalkin";
+    repo = "pytest-leaks";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Oxjal/bkgynFiYubFABwbNJooGx9GwxQwPy5BLI2DEY=";
+  };
+
+  build-system = [
+    setuptools
+    setuptools-scm
+    wheel
+  ];
+
+  dependencies = [ pytest ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  disabledTests = [
+    # assert 0
+    "test_broken_faucet"
```

Comment:
Is this a flaky test? A known failure? An invalid pre-condition?

Reply:
Not sure